runtime/doc/tabpage.txt

```diff
@@ -191,8 +222,10 @@ clarification what +N means in this context see |[range]|.
 LOOPING OVER TAB PAGES:
 
 							*:tabd* *:tabdo*
-:tabd[o] {cmd}	Execute {cmd} in each tab page.
-		It works like doing this: >
+:[range]tabd[o] {cmd}
+        	Execute {cmd} in each tab page or if [range] is given only in
```

Comment:
Maybe

~~~
		Execute {cmd} in each tab page or, if [range] is given, only
		in tabpages which tab page number is in the [range]. It works
~~~